### DISCUSSION TITLE
Error: WebSocket close with status code 1006

### DISCUSSION THREAD

**Discussion by adrien914** [coder/code-server#4410] [Q&A] [answered]
Hello tech friends !

I'm having a weird issue that i can't manage to fix, i installed a code-server on a debian 10 cloud instance and I can't seem to be able to connect to it, i'm getting the error you see on the image below after.

![image](https://user-images.githubusercontent.com/43916731/139257954-98741db0-07d2-47f0-a529-fbe57cdd1188.png)

I'm using the latest release of code-sever and am using apache2 as a reverse proxy, i am using https but i also tried with http and it doesn't work either so i don't think the issue comes from this.
I saw some posts saying i should disable a proxy to localhost thing in the settings.json but i can't find this file anywhere.
I tried to reinstall it, to run it on 0.0.0.0 instead of 127.0.0.1 and nothing works.

Anyone has an idea what's causing this ?

**Comment by jsjoeio** [comment]
This is very odd indeed! Are there any browser/server logs you can share? That will help me or @code-asher help you!

**Comment by code-asher** [comment, marked as answer]
Make sure you have set up apache to handle websockets.

https://github.com/cdr/code-server/issues/1544#issuecomment-743912980

We still need to add Apache to the docs.

https://github.com/cdr/code-server/issues/2452

**Reply by adrien914** [reply]
Hey, sorry for not answering earlier it's been a very busy day at work today. I'll see about setting up apache to handle we sockets i haven't done it so it must be what's causing the issue. Thanks a lot to you both.

**Reply by adrien914** [reply]
It works with the linked apache configuration, thanks again !

**Reply by jakob-huber** [reply]
I had the same issue with code server running in a container with a local domain and reverse proxy. The apache configuration above solved the issue. Thanks
